client/asset/index.html

```diff
@@ -5,7 +5,6 @@
   <div id='overview'></div>
 </body>
 <script type="module" src='./overview.js'></script>
-<!-- <script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
-<script crossorigin src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
-<script src="https://unpkg.com/axios/dist/axios.min.js"></script> -->
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```

Comment:
I would love to know how this works!!!

Reply:
https://fontawesome.com/v5.15/how-to-use/customizing-wordpress/snippets/setup-cdn-webfont

I used this article but basically it's just the public CDN for the font-awesome file so that you dont have to npm install all that stuff i didnt npm install..